components/viz/service/display/overlay_processor_android.cc

```diff
@@ -60,7 +61,13 @@ OverlayProcessorAndroid::OverlayProcessorAndroid(
   // a dummy resource that has no relation to what the overlay contains.
   // https://crbug.com/842931 .
 #if BUILDFLAG(USE_STARBOARD_MEDIA)
-  strategies_.push_back(std::make_unique<OverlayStrategyUnderlayStarboard>(this));
+  if (media::IsStarboardRendererEnabled()) {
```

Comment:
nit: Can we use

```
#if BUILDFLAG(USE_STARBOARD_MEDIA)
  if (media::IsStarboardRendererEnabled()) {
    strategies_.push_back(
        std::make_unique<OverlayStrategyUnderlayStarboard>(this));
  }
#endif   // BUILDFLAG(USE_STARBOARD_MEDIA)
```

and then fallback to the block below (and remove the #else)?